lib/configs/_base.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
         "init-declarations": "error",
         "linebreak-style": ["error", "unix"],
         "lines-between-class-members": "error",
+        "logical-assignment-operators": "off", // TODO: enable once we drop ESLint v7 support
```

Comment:
@eslint-community/mysticatea-eslint-plugin I added the new ESLint v8 rules as `'off'` as adding them would be a breaking change

They're also not available when users of this plugin don't have v8 installed

Reply:
I think it's probably best to change the test case and not add the rule option.
Users may unintentionally turn off a rule because `"off"` overrides the previous configuration of `extends`.

Reply:
Can you elaborate?

Making this an error, would create possible more ESLint errors, so this is a breaking change according to our semver policy.

Also: people that don't have ESLint v8 won't have these rules available, so setting them to error will make ESLint mad.

This has nothing to do with our test cases?

Reply:
> Can you elaborate?

I want to remove the line that turns off the rule.

```suggestion
```

> This has nothing to do with our test cases?

It may have been my misunderstanding.
I thought there was a test that would fail if we didn't set the rule to `off` or `error`.

> Making this an error, would create possible more ESLint errors, so this is a breaking change according to our semver policy.

I don't think it's against policy to turn it off.
However, I thought that removing the rule options would make the configuration more user-friendly.

Reply:
> I thought there was a test that would fail if we didn't set the rule to `off` or `error`.

This is indeed the case.
The tests are testing that we have all available rules available in our configs

Reply:
What do you think about getting `coreRules` information in our tests like this:

```js
const cp = require("child_process")
const semver = require("semver")
const eslintMinimumSupportedVersion = semver.minVersion(
    require("../../../package.json").peerDependencies.eslint
).version
// eslint-disable-next-line @eslint-community/mysticatea/node/no-sync -- for test
cp.execSync(
    `npm install --no-save @eslint-community/eslint-minimum-supported@npm:eslint@${eslintMinimumSupportedVersion}`,
    {
        stdio: "inherit",
    }
)
// eslint-disable-next-line @eslint-community/mysticatea/node/no-missing-require -- for test
const { Linter } = require("@eslint-community/eslint-minimum-supported")
const coreRules = new Linter().getRules()
// eslint-disable-next-line @eslint-community/mysticatea/node/no-sync -- for test
cp.execSync(
    "npm uninstall --no-save @eslint-community/eslint-minimum-supported",
    {
        stdio: "inherit",
    }
)
```

Reply:
@ota-meshi I used the same method as I've done in #43